sihl-user/src/sihl_user.ml

```diff
@@ -213,6 +213,14 @@ module Make (Repo : User_repo.Sig) : Sihl.Contract.User.Sig = struct
     Repo.register_cleaner ();
     Sihl.Container.Service.create ~commands:[ create_admin_cmd ] lifecycle
   ;;
+
+  module Web = struct
+    let user_from_session =
+      Web.user_from_session (fun user_id -> find_opt ~user_id)
```

Comment:
Is there an issue to remove all named arguments where they aren't needed? Could've eta-reduced this lambda :sunglasses:

Reply:
I don't want to break the user service API in this change, but we should change it in a later release.

Reply:
Yea, you're right, smart.